src/database/queries/entity.js

```diff
@@ -0,0 +1,64 @@
+const Entity = require('../models/index').Entity
+module.exports = class UserEntityData {
+	static async createEntity(data) {
+		try {
+			return await Entity.create(data, { returning: true })
+		} catch (error) {
+			console.log(error)
+			throw error
+		}
+	}
+
+	static async findAllEntities(filter) {
+		try {
+			filter.status = 'ACTIVE'
```

Comment:
This seems like a very restrictive query definition which overrides the caller's filter. This can lead to unexpected results when using this query elsewhere. This is a fair override only if the query was findAllActiveEntities. [DISCUSS]

Reply:
resolved [here](https://github.com/ELEVATE-Project/mentoring/pull/258/files#diff-6de8a4314ea72d104613372fdbc910ffec808545d44a776c725158d0ba779734).